javascript-sdk/examples/run-script-in-headless-browser/run-script-in-headless-browser.js

```diff
@@ -0,0 +1,47 @@
+/* This example demonstrates how to run the script in a headless browser. */
+
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+
+// Define the URL of the page to scrape.
+const URL = 'https://scrapeme.live/shop/';
+
+// Define the queries to locate the search box and fetch the stock number.
+const SEARCH_QUERY = `
+{
+    search_products_box
+}
+`;
+
+const STOCK_NUMBER_QUERY = `
+{
+    number_in_stock
+}
+`;
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  configure({ apiKey: process.env.AGENTQL_API_KEY });
```

Comment:
We have decided to remove this line from example. It will be included in the installation guide.